scripts/kibana/validate-ts-interfaces-against-apm-server-sample-docs/scripts/download-sample-docs.ts

```diff
@@ -17,17 +17,17 @@ interface DocType {
 const docTypes: DocType[] = [
   {
     interfaceName: 'SpanRaw',
-    interfacePath: '../apm-ui-interfaces/raw/SpanRaw',
+    interfacePath: '../apm-ui-interfaces/raw/span_raw',
```

Comment:
Master branch name file:
- https://github.com/elastic/kibana/tree/master/x-pack/plugins/apm/typings/es_schemas/raw

7.6 branch name file:
- https://github.com/elastic/kibana/tree/7.6/x-pack/legacy/plugins/apm/typings/es_schemas/raw

I've no clue if it's possible to support different interfaces at the same time.

Otherwise, the changes in master are breaking changes in this particular implementation. No clue how to move forward